src/flowTypes.ts

```diff
@@ -299,8 +299,10 @@ export interface Flow {
 }
 
 export interface Action {
+  [x: string]: any;
```

Comment:
What is the meaning of "`x`", is this the most descriptive name that could be used here?